config/routes.rb

```diff
@@ -131,6 +131,14 @@
     end
   end
 
+  if defined?(Sufia::StatsAdmin)
+    namespace :admin do
+      constraints Sufia::StatsAdmin do
```

Comment:
Not saying you have to, but you can do `can :show, :stats if user.admin?` https://github.com/ryanb/cancan/wiki/Non-RESTful-Controllers